client/login/wp-login/index.jsx

```diff
@@ -465,7 +468,10 @@ export class Login extends Component {
 			return this.renderGravPoweredLoginBlockFooter();
 		}
 
-		if ( currentQuery.lostpassword_flow === 'true' ) {
+		if (
+			currentQuery.lostpassword_flow === 'true' &&
+			config.isEnabled( 'woocommerce/core-profiler-passwordless-auth' )
```

Comment:
This also may impact other flows since it's not bounded to `isWooCoreProfilerFlow`

Reply:
@ilyasfoo Good catch 👍 
Fixed in [Tighten passwordless auth check with isWooCoreProfilerFlow](https://github.com/Automattic/wp-calypso/pull/94268/commits/085d1d61824519b2ff7a287258c8d05a153024bf)